src/user/organization/popups/Members.js

```diff
@@ -202,9 +202,9 @@ class Members extends Component {
               </div>
               <div className="share__btn__container">
                 <p className="share__text">or share invite on</p>
-                <Button className="invite__btn">Facebook</Button>
-                <Button className="invite__btn">LinkedIn</Button>
-                <Button className="invite__btn">Twitter</Button>
+                <Button className="invite__btn" target = "_blank" style={{"padding-top":"6px"}} href={"https://www.facebook.com/sharer/sharer.php?u=" + inviteLink}>Facebook</Button>
```

Comment:
Hi @pranshukharkwal 
Can  you please add preview links in the form of cards when sharing?
cc @jaskiratsingh2000

Reply:
Hi @Rupeshiya 
I am not exactly sure what you mean by preview links? Can you please explain? Or show an example?

Reply:
I mean like when we use any link for sharing on LinkedIn or Twitter or any social media it shows the preview of the links. When we are sharing.

Reply:
So you want preview for the Invite Link ( which would essentially be the Donut Homepage), or for the sharing link of that particular social media?

Reply:
Also I am thinking to use facebook graph API for link preview. What do you think? Should I go for it or you would suggest some other method?